### DISCUSSION TITLE
Feat: Wrapped 2023

### DISCUSSION THREAD

**Discussion by bogdaaamn** [aws/codewhisperer-command-line-discussions#42] [Feature request]
### Checks

- [X] I could not find an existing setting that solved my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my feature request

### Description

I know this might be silly, but I really enjoyed the previous year's `fig wrapped`. Will there be a `cw wrapped` similar with what we used to have?

**Comment by chaynabors** [comment]
Sorry for responding so late on this @bogdaaamn. I wrote fig wrapped last year. Fig wrapped in prior years used the shell history on your machine in order for us to show those analytics. There's no remote DB or anything for us to migrate in this case (as there shouldn't be) so we didn't have any good user shell history to 'wrap' on. Wrapped in 2024 isn't confirmed internally or otherwise at this point but I have a pretty good feeling about it personally.
